install-dev/upgrade/sql/1.7.8.0.sql

```diff
@@ -0,0 +1,10 @@
+SET SESSION sql_mode='';
+SET NAMES 'utf8';
+
+INSERT IGNORE INTO `PREFIX_hook` (`id_hook`, `name`, `title`, `description`, `position`) VALUES
+  (NULL, 'displayFooterCategory', 'Category footer', 'This hook adds new blocks under the products listing in a category/search', '1 '),
+  (NULL, 'actionPresentCart', 'Cart Presenter', 'This hook is called before a cart is presented', '1 '),
```

Comment:
I just noticed, I believe we made a typo error here (the same can be found in upgrade scripts up to `1.7.4.0` 😅)
The position inserted is `'1 '` (string with extra space), I guess mysql or the sql interpreter is smart enough to set it correctly but still it would be better to simply use an int `1` What do you think?

Reply:
I thought it was on purpose 😄 also indeed I was wondering what the hell it was

Well, I told myself "must be a legacy rule ..."

Reply:
Well I can't be sure but it seems like a mistake, especially since the database field is a TINYINT

Reply:
Fixed